Calculator.py

```diff
@@ -17,6 +17,9 @@ def run_operation(number_1,  number_2, operation):
     elif operation == '/':
         print('{} / {} = '.format(number_1, number_2), end = '')
         return number_1 / number_2
+    elif operation == '%':
+        print('{} % {} = '.format(number_1, number_2), end = '')
```

Comment:
Print statement looks great and you added the correct corresponding elif

Reply:
I agree, the print statement is correctly formatted and should function as intended

Reply:
One thing i think we could try is adding the modulo sign in the beginning input stage, as well as maybe a space so users can easily see what they type and the options they have available